### DISCUSSION TITLE
چگونه تغییر مقدار اینپوت را رصد کنیم؟

### DISCUSSION THREAD

**Discussion by hushmand12** [majidh1/JalaliDatePicker#41] [Question] [answered]
سلام دوست عزیز، چگونه تغییر مقدار اینپوت را رصد کنیم؟
رویداد onChange روی چی باید نوشته بشه؟
میخوام بعد از انتخاب تاریخ یه پیام به کاربر بدم
لطفا راهنمایی کنید با تشکر

**Comment by majidh1** [comment, marked as answer]
سلام، به وسیله ایونت Change میتونید از تغییر اینپوت با خبر بشید

```
<input type="text" data-jdp id="change_sample">
document.getElementById('change_sample').addEventListener('change', (event) => {
  console.log(event)
});
```
